src/Symfony/Component/Form/Form.php

```diff
@@ -946,7 +946,7 @@ public function get(string $name)
      *
      * @return bool
      */
-    public function offsetExists($name)
+    public function offsetExists(mixed $name)
```

Comment:
Why don't we use `int|string` for array access? If we can't I'd say it is worth a deprecation in 5.4 to do so in Symfony 6.

Reply:
The ArrayAccess interface mandates using `mixed` in all its methods, no way around.